### ISSUE TITLE
Something for both devs and users to consider...  bonuses

### ISSUE THREAD

**Issue by ChezLinux** [finndev/PokeBuddy#443]
Not trying to scare anyone, but you may want to consider dropping the +100 "Excellent" xp bonus for catching Pokemon.  Considering the "Nice", "Great" & "Excellent" bonuses are not currently in the live apps, it will make it quite easy for Niantic to flag accounts using any bots that use it.  The curve bonus still exists, but it wouldn't take much to do a database query to filter accounts who received any "Excellent" bonus and drop the banhammer

Just a heads up.

**Comment by naor173** [comment]
you are right i think im screwed... look at my messege man tell me what you think?

**Comment by pokemonloverx333** [comment]
its still an illegal bot , what did you think , that we get pass with that ? i just use bot for fun , wanna see how far i get and which pokemons i get , i also have an regular main where i only play correctly , its illegal and you will be busted for it , what u thought will happen?

**Comment by flixwito** [comment]
Yeah, nice idea.
